test/jest/mparticle-instance-manager.test.ts

```diff
@@ -0,0 +1,27 @@
+import Constants from '../../src/constants';
+import { IMParticleInstanceManager } from '../../src/sdkRuntimeModels';
+const mockPerformanceMark = (globalThis as any).__mockPerformanceMark;
+
+
+const mParticle = (globalThis as any).mParticle as IMParticleInstanceManager;
+
+describe('mParticle instance manager', () => {
+
+    it('sets sdkStart event timing', async () => {
+        expect(mockPerformanceMark).toHaveBeenCalledWith(Constants.PerformanceMetricsNames.SdkStart);
+    });
+
+    it('generates a launcher instance guid', () => {
+        const guid = mParticle.getLauncherInstanceGuid();
+        expect(guid).toBeDefined();
```

Comment:
i think even if you had 
const guid = null
that expect(guid).toBeDefined(); would pass. It's just checking to see if it's defined or not. 

I'd advocate for 

expect(typeof guid).toBe('string');